examples/reference.yml

```diff
@@ -225,3 +225,10 @@ acl_mongo:
   # the MongoDB server.
   # (See https://golang.org/pkg/time/#ParseDuration for a format description.)
   cache_ttl: "1m"
+
+# External authorization - call an external progam to authorize user.
+# JSON of authz.AuthRequestInfo is passed to command's stdin and exit code is examined.
+# 0 - allow, 1 - deny, other - error.
+acl_ext:
```

Comment:
since it doesn't provide external _acls_, but external _authz_, let's call it `ext_authz`.

i have been thinking on sourcing acls externally as well. perhaps `acl_ext` should become an umbrella for acl sources, the thing you asked for in the first place, plus caching and refresh mechanism. so, `acl_ext` would configure one or more sources of acls (mongo, mysql, static file, ...), poll and cache them. but that's another story. let's keep this PR just about authz for now.